Cargo.toml

```diff
@@ -22,9 +22,9 @@ hyper = "^0.10"
 hyper-openssl = { version = "^0.2", optional = true }
 log = "^0.3"
 maplit = "^0.1"
```

Comment:
Ignore this comment, we still use it in a couple of other places, but they should be gone soon.